tutorials/29_Serializing_Pipelines.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
<p>There's a typo in the component name, extra "a" after "..Gene". Correct version -&gt;  <code><a href="https://docs.haystack.deepset.ai/v2.0/docs/huggingfacelocalgenerator" target="_blank">HuggingFaceLocalGenerator</a></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/272/discussion/#comment-1471446395'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>